src/test/ovmsconfig_test.cpp

```diff
@@ -329,6 +330,609 @@ TEST_F(OvmsConfigDeathTest, negativeMissingDashes) {
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.modelName, servingName);
+}
+
+TEST(OvmsGraphConfigTest, positiveDefaultEmbeddings) {
+    std::string modelName = "OpenVINO/Phi-3-mini-FastDraft-50M-int8-ov";
+    std::string downloadPath = "test/repository";
+    char* n_argv[] = {
+        (char*)"ovms",
+        (char*)"--pull",
+        (char*)"--source_model",
+        (char*)modelName.c_str(),
+        (char*)"--model_repository_path",
+        (char*)downloadPath.c_str(),
+        (char*)"--task",
+        (char*)"embeddings",
+    };
+
+    int arg_count = 8;
+    ConstructorEnabledConfig config;
+    config.parse(arg_count, n_argv);
+
+    auto& hfSettings = config.getServerSettings().hfSettings;
+    ASSERT_EQ(hfSettings.sourceModel, modelName);
+    ASSERT_EQ(hfSettings.downloadPath, downloadPath);
+    ASSERT_EQ(hfSettings.pullHfModelMode, true);
+    ASSERT_EQ(hfSettings.task, ovms::embeddings);
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.normalize, "false");
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.truncate, "false");
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.version, 1);
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.numStreams, 1);
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.targetDevice, "CPU");
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.modelName, modelName);
+}
+
+TEST(OvmsGraphConfigTest, positiveSomeChangedEmbeddings) {
+    std::string modelName = "OpenVINO/Phi-3-mini-FastDraft-50M-int8-ov";
+    std::string downloadPath = "test/repository";
+    std::string servingName = "FastDraft";
+    char* n_argv[] = {
+        (char*)"ovms",
+        (char*)"--pull",
+        (char*)"--source_model",
+        (char*)modelName.c_str(),
+        (char*)"--model_repository_path",
+        (char*)downloadPath.c_str(),
+        (char*)"--model_version",
+        (char*)"2",
+        (char*)"--task",
+        (char*)"embeddings",
+        (char*)"--graph_target_device",
+        (char*)"GPU",
+        (char*)"--normalize",
+        (char*)"true",
+        (char*)"--model_name",
+        (char*)servingName.c_str(),
+    };
+
+    int arg_count = 16;
+    ConstructorEnabledConfig config;
+    config.parse(arg_count, n_argv);
+
+    auto& hfSettings = config.getServerSettings().hfSettings;
+    ASSERT_EQ(hfSettings.sourceModel, modelName);
+    ASSERT_EQ(hfSettings.downloadPath, downloadPath);
+    ASSERT_EQ(hfSettings.pullHfModelMode, true);
+    ASSERT_EQ(hfSettings.task, ovms::embeddings);
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.version, 2);
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.numStreams, 1);
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.normalize, "true");
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.truncate, "false");
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.targetDevice, "GPU");
+    ASSERT_EQ(hfSettings.embeddingsGraphSettings.modelName, servingName);
+}
+
+TEST(OvmsGraphConfigTest, modelSettings) {
```

Comment:
```suggestion
TEST(OvmsGraphConfigTest, ensureModelNameAndPathSetForHfSettings) {
```
That is the purpose of this test correct?